test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAnAppWithLibrary.cs

```diff
@@ -128,7 +128,9 @@ public void It_generates_satellite_assemblies()
                 {"", "Welcome to .Net!"},
                 {"da", "Velkommen til .Net!"},
                 {"de", "Willkommen in .Net!"},
-                {"fr", "Bienvenue à .Net!"}
+                //  Only check partial French output due to Console encoding issue: https://github.com/dotnet/cli/issues/6597
+                {"fr", "Bienvenue"}
+                //{"fr", "Bienvenue à .Net!"}
```

Comment:
Makes sense.  Any thoughts on whether we should register the code pages provider in the test app or just retarget it to .NET Core 2.0?